src/app/canvas/qgsappcanvasfiltering.cpp

```diff
@@ -40,6 +41,18 @@ void QgsAppCanvasFiltering::setupElevationControllerAction( QAction *action, Qgs
       {
         QgisApp::instance()->showProjectProperties( tr( "Elevation" ) );
       } );
+      QAction *setRangeSizeAction = new QAction( tr( "Set Fixed Range Size…" ), controller );
+      controller->menu()->addAction( setRangeSizeAction );
+      connect( setRangeSizeAction, &QAction::triggered, QgisApp::instance(), [controller ]
+      {
+        const double existingSize = controller->fixedRangeSize();
+        QgsElevationControllerFixedSizeDialog dialog( controller );
+        dialog.setFixedRangeSize( existingSize >= 0 ? existingSize : -1 );
+        if ( dialog.exec() )
+        {
+          controller->setFixedRangeSize( dialog.fixedRangeSize() );
+        }
+      } );
```

Comment:
It would be probably nicer if we had the fixed range size configuration embedded in the menu, like you did it for 2d map canvas dock widgets (e.g. scale, rotation) instead of having a separate modal input dialog.

Thinking about it more, it nearly asks for a full styling panel kind of configuration widget, especially if we wanted more configuration options in the future (fixed steps for the range slider etc.) :smile:

Reply:
> It would be probably nicer if we had the fixed range size configuration embedded in the menu, like you did it for 2d map canvas dock widgets (e.g. scale, rotation) instead of having a separate modal input dialog.

OK, done!

> Thinking about it more, it nearly asks for a full styling panel kind of configuration widget, especially if we wanted more configuration options in the future (fixed steps for the range slider etc.) 😄

Let's defer for now :wink: